fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeMetadata.java

```diff
@@ -197,7 +196,7 @@ private void collectDeltaLakePlanFiles(PredicateSearchKey key, Table table, Scal
                         schema.get(column).getDataType().toString())
                         && !partitionColumns.contains(column)).collect(toImmutableList());
 
-        List<Row> files = Lists.newArrayList();
+        List<FileScanTask> files = Lists.newArrayList();
 
         try (CloseableIterator<FilteredColumnarBatch> scanFilesAsBatches = scan.getScanFiles(engine, true)) {
```

Comment:
```suggestion
        try (CloseableIterator<FilteredColumnarBatch> scanFilesAsBatches = scan.getScanFiles(engine, enableCollectColumnStatistics(connectContext))) {
```

Reply:
This must be true because the row number information is in the sson of stats.